tests/validation/tests/v3_api/test_windows_cluster.py

```diff
@@ -8,36 +7,40 @@
 from .test_rke_cluster_provisioning import create_custom_host_from_nodes
 from .test_rke_cluster_provisioning import HOST_NAME
 
-from lib.aws import AmazonWebServices
-from threading import Thread
+from lib.aws import AmazonWebServices, AWS_DEFAULT_AMI, AWS_DEFAULT_USER
+
 
 def test_windows_provisioning():
     node_roles_linux = [["controlplane"], ["etcd"], ["worker"]]
     node_roles_windows = [["worker"], ["worker"], ["worker"]]
 
     win_nodes = \
         AmazonWebServices().create_multiple_nodes(
-            len(node_roles_windows), random_test_name(HOST_NAME), os_version="windows-1903", 
-            docker_version="19.03")
+            len(node_roles_windows), random_test_name(HOST_NAME))
 
-    aws_nodes = \
+    linux_nodes = \
         AmazonWebServices().create_multiple_nodes(
-            len(node_roles_linux), random_test_name(HOST_NAME))
+            len(node_roles_linux), random_test_name(HOST_NAME),
+            ami=AWS_DEFAULT_AMI, ssh_user=AWS_DEFAULT_USER)
```

Comment:
What's the reason we're using `AWS_DEFAULT_AMI` and `AWS_DEFAULT_USER` instead of `AWS_AMI` and `AWS_DEFAULT_USER` here ?

Reply:
when running `test_windows_provisioning`  `AWS_AMI` and `AWS_USER` are values for the Windows nodes. 
we don't want to introduce too many parameters in the Jenkins job so make the default values for Linux and use them here.

Reply:
Makes sense for the Windows cluster test. Thanks @jiaqiluo